tfhe/src/shortint/ciphertext/mod.rs

```diff
@@ -1,897 +1,9 @@
-        let data = [
-            (Degree(3), Degree(3), Degree(3)),
-            (Degree(3), Degree(1), Degree(3)),
-            (Degree(1), Degree(3), Degree(3)),
-            (Degree(3), Degree(2), Degree(3)),
-            (Degree(2), Degree(3), Degree(3)),
-            (Degree(2), Degree(2), Degree(3)),
-            (Degree(2), Degree(1), Degree(3)),
-            (Degree(1), Degree(2), Degree(3)),
-            (Degree(1), Degree(1), Degree(1)),
-            (Degree(0), Degree(1), Degree(1)),
-            (Degree(0), Degree(1), Degree(1)),
-        ];
-
-        for (lhs, rhs, expected) in data {
-            let result = lhs.after_bitxor(rhs);
-            assert_eq!(
-                result, expected,
-                "For a bitxor between variables of degree {lhs:?} and {rhs:?},\
-             expected resulting degree: {expected:?}, got {result:?}"
-            );
-        }
-    }
-    #[test]
-    fn degree_after_bitor_ci_run_filter() {
-        let data = [
-            (Degree(3), Degree(3), Degree(3)),
-            (Degree(3), Degree(1), Degree(3)),
-            (Degree(1), Degree(3), Degree(3)),
-            (Degree(3), Degree(2), Degree(3)),
-            (Degree(2), Degree(3), Degree(3)),
-            (Degree(2), Degree(2), Degree(3)),
-            (Degree(2), Degree(1), Degree(3)),
-            (Degree(1), Degree(2), Degree(3)),
-            (Degree(1), Degree(1), Degree(1)),
-            (Degree(0), Degree(1), Degree(1)),
-            (Degree(0), Degree(1), Degree(1)),
-        ];
-
-        for (lhs, rhs, expected) in data {
-            let result = lhs.after_bitor(rhs);
-            assert_eq!(
-                result, expected,
-                "For a bitor between variables of degree {lhs:?} and {rhs:?},\
-             expected resulting degree: {expected:?}, got {result:?}"
-            );
-        }
-    }
-
-    #[test]
-    fn degree_after_bitand_ci_run_filter() {
-        let data = [
-            (Degree(3), Degree(3), Degree(3)),
-            (Degree(3), Degree(1), Degree(1)),
-            (Degree(1), Degree(3), Degree(1)),
-            (Degree(3), Degree(2), Degree(2)),
-            (Degree(2), Degree(3), Degree(2)),
-            (Degree(2), Degree(2), Degree(2)),
-            (Degree(2), Degree(1), Degree(1)),
-            (Degree(1), Degree(2), Degree(1)),
-            (Degree(1), Degree(1), Degree(1)),
-            (Degree(0), Degree(1), Degree(0)),
-            (Degree(0), Degree(1), Degree(0)),
-        ];
-
-        for (lhs, rhs, expected) in data {
-            let result = lhs.after_bitand(rhs);
-            assert_eq!(
-                result, expected,
-                "For a bitand between variables of degree {lhs:?} and {rhs:?},\
-             expected resulting degree: {expected:?}, got {result:?}"
-            );
-        }
-    }
-}
+mod common;
+mod compact_list;
+mod compressed;
+mod standard;
+
+pub use common::*;
+pub use compact_list::*;
+pub use compressed::*;
+pub use standard::*;
```

Comment:
let's not have pub use of everything, it was a bad pattern/practice in core_crypto, Thomas said it was not the best way of making things available where required

Reply:
Removed